core/dtn7/src/bin/dtnd.rs

```diff
@@ -118,6 +124,19 @@ async fn main() -> Result<(), std::io::Error> {
                 .help("Sets web interface port (default = 3000)")
                 .takes_value(true),
         )
+        .arg(
+            Arg::new("eclatcpport")
```

Comment:
- Good point about WebSockets
- Regarding ``-C``. As the ECLA is not directly initialised as CLA the ``-C`` options doesn't really apply. The ECLA is an own system so to speak that in itself dynamically creates and removes CLA's e.g. creates ``ExternalConvergenceLayer`` with ``name`` and optional ``port`` settings.
- I still acknowledge that the current arguments might be suboptimal and a better way of handling it could be found